advanced/Templates/gravity.db.sql

```diff
@@ -48,11 +83,12 @@ INSERT INTO info VALUES("version","1");
 
 CREATE VIEW vw_gravity AS SELECT a.domain
     FROM gravity a
-    WHERE a.domain NOT IN (SELECT domain from whitelist WHERE enabled == 1);
+    WHERE a.domain NOT IN (SELECT domain from vw_whitelist);
 
 CREATE VIEW vw_whitelist AS SELECT a.domain
     FROM whitelist a
-    WHERE a.enabled == 1
+    INNER JOIN domain_groups b ON b.id = a.group_id
```

Comment:
For readability it would be best to use better aliases for the tables, or just use the full table names.

Reply:
I will remove the aliases.